src/Form/SummaryType.php

```diff
@@ -47,7 +68,34 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ->add('availabilities', JobDurationChoiceType::class)
                     ->add('job_locations', JobLocationChoiceType::class)
                     ->add('professional_synopsis', TextareaType::class)
-                ;
+                    ->add('profilePicture', FileType::class, [
+                        'required' => false,
+                        'label' => false,
+                    ]);
+
+                $builder
+                    ->addEventListener(FormEvents::POST_SUBMIT, function (FormEvent $event) {
+                        if (!$event->getForm()->isValid()) {
+                            return;
+                        }
+
+                        /** @var Summary $summary */
+                        $summary = $event->getData();
+                        // Save profile picture to cloud storage
+                        if ($summary->getProfilePicture()) {
```

Comment:
We should enter that if only when this is an instance of `UploadedFile`. Please add a check for it.